### PR TITLE
Replace Get Help community links with a single one

### PR DESCRIPTION
**Pull request by 4ldas**
Added the official Rust programming language Discord server to the "Getting Help" section

### CODE REVIEW

**Comment by mark-i-m** [comment]
I believe the discord was explicitly removed in the past because T-compiler generally prefers zulip. Not sure about libs, rustdoc, infra, etc teams, though.

**Comment by faith** [comment]
> I believe the discord was explicitly removed in the past because T-compiler generally prefers zulip. Not sure about libs, rustdoc, infra, etc teams, though.

Oh well they got a official discord server for users, not only the developer team so why not include 🤷‍♂️

**Comment by LeSeulArtichaut** [comment]
Also, this is the Getting Help section which is not about the compiler team in particular. I think adding the official Discord server, and even the community server and IRLO, makes sense.

**Comment by Mark-Simulacrum** [comment]
Hm, so I think this PR draws my attention to this section and I'm wondering if the better thing might be to link to https://www.rust-lang.org/community? That feels like a more appropriate place and more likely to be kept up to date.

**Comment by JohnCSimon** [comment]
Ping from triage
@4ldas - can you please address the comment from Mark-Simulacrum or if you have already, please alert the reviewers so this can be reviewed and merged. Thank you!

**Comment by JohnCSimon** [comment]
Ping again from triage  - @4ldas

**Comment by 4ldas** [comment]
> Hm, so I think this PR draws my attention to this section and I'm wondering if the better thing might be to link to https://www.rust-lang.org/community? That feels like a more appropriate place and more likely to be kept up to date.

Then i guess the getting help shouldnt be a section but a hyperlink i guess

**Comment by Mark-Simulacrum** [comment]
Yes, I think a short paragraph (maybe 1-2 sentences) would make sense. Would you be up for changing the text along those lines? I think we should aim to just point at https://www.rust-lang.org/community and say that there's resources there for help, discussion, and more or something like that.

**Comment by Mark-Simulacrum** [comment]
cc @rust-lang/community as well in case folks have alternative strategies to suggest for how to guide people people seeking help

**Comment by crlf0710** [comment]
@4ldas Ping from triage, any updates here?

**Comment by 4ldas** [comment]
took ages but here it is hehe

File: README.md [hunk 1]
```diff
@@ -226,15 +226,7 @@ supported build environments that are most likely to work.
...
-The Rust community congregates in a few places:
-
-* [Stack Overflow] - Direct questions about using the language.
-* [users.rust-lang.org] - General discussion and broader questions.
-* [/r/rust] - News and general discussion.
-
-[Stack Overflow]: https://stackoverflow.com/questions/tagged/rust
-[/r/rust]: https://reddit.com/r/rust
-[users.rust-lang.org]: https://users.rust-lang.org/
+The Rust community congregates in a few places, you can find them at the [community tab of the Rust website](https://www.rust-lang.org/community)
```
Comment (JohnTitor) [inline README.md:229]:
```suggestion
The Rust community congregates in a few places, you can find them
at the [community tab of the Rust website](https://www.rust-lang.org/community).
```

Comment (4ldas) [reply README.md:229]: Don't see much of a point in this

Comment (JohnTitor) [reply README.md:229]: You forgot to add `.`, the line length is a bonus point.

**Comment by crlf0710** [comment]
@4ldas Ping from triage, what's the current status of this? Could you address the comments above, and mark this as ready for review (with the rustbot) when you are done?

**Comment by JohnCSimon** [comment]
@4ldas Ping again from triage, what's the current status of this?

**Comment by crlf0710** [comment]
@4ldas Ping from triage, any updates?

**Comment by JohnCSimon** [comment]
Closing this as inactive

**Pull request closed without merging**